content/guides/spring/spring-route-authorization-testing.md

```diff
@@ -0,0 +1,550 @@
+---
+draft: false
+date: 2021-05-05T10:40:17-06:00
+tags: ["spring","testing", "spring security"]
+title: "Spring Route Authorization Testing"
+description: >
+    Dynamic testing of your Spring Security route authorizations
+date: "2021-05-12"
+topics:
+- Spring
+- Testing
+tags:
+- Spring
+- Spring Security
+- Spring Testing
+# Author(s)
+team: 
+- Toby Rumans
```

Comment:
We have the ability to make this link to a small bio page with a headshot, if you're interested in adding that.

Reply:
Sounds good to me

Reply:
So you would just create a `toby-rumans` directory under `content/team`. Include as much info as you'd like - the existing entries there have lots of examples of the various fields that are available.

Reply:
done, thanks Zac